### PR TITLE
Fix only code style issues in language and package models

### PR DESCRIPTION
**Pull request by javigomez**
related to issue #130

### CODE REVIEW

**Comment by infograf768** [comment]
Thanks

File: component/admin/models/package.php [hunk 1]
```diff
@@ -344,7 +345,6 @@ public function save($data)
 			$manifestElement = $dom->createElement('manifest', $manifest);
 			$versionElement = $dom->createElement('version', $data['version']);
 			$authorElement = $dom->createElement('author', $data['author']);
-			$copyrightElement = $dom->createElement('copyright', $data['copyright']);
```
Comment (wilsonge) [inline component/admin/models/package.php:347]: Should this have been removed?

Comment (javigomez) [inline component/admin/models/package.php:347]: yes, is repeated just a few lines after. It was ovewritten.

Comment (javigomez) [inline component/admin/models/package.php:347]: specifically here https://github.com/joomla-projects/com_localise/pull/131/files#diff-aabe745fa672e9b0deb72a3c70e71525R352

Comment (wilsonge) [inline component/admin/models/package.php:347]: Oh yeah :) my bad!

Comment (javigomez) [inline component/admin/models/package.php:347]: no no, your good. Is super great that we review others pulls ^_^

Comment (wilsonge) [inline component/admin/models/package.php:347]: https://github.com/joomla-projects/com_localise/pull/137 we should do this then as well for same reason

**Pull request merged**